src/containers/forms/CommentsForm.jsx

```diff
@@ -63,6 +68,19 @@ class CommentsForm extends Component {
     };
   }
 
+  /**
+   * @function componentDidUpdate
+   * @summary Closes model after showing success message
+   * @param {Object} prevProps
+   */
+  componentDidUpdate(prevProps) {
+    if (prevProps.message.type !== this.props.message.type) {
```

Comment:
Would using `&&` to combine this conditions still work? If so, that would be the way to got avoid nesting